packages/hpe-icons/README.md

```diff
@@ -1,69 +1,137 @@
+- **Accessible**: Built with accessibility best practices
+- **Customizable**: Full control over size, color, and styling
+
+## Installation
+
+```bash
+npm install hpe-icons
+# or
+yarn add hpe-icons
+```
+
+## Usage
+
+### Grommet Integration
+
+The easiest way to use HPE icons is with the Grommet integration, which provides pre-styled React components with built-in theming support:
+
+```tsx
+import { Grommet } from 'grommet';
+import { hpe } from 'grommet-theme-hpe';
+import { Accessibility, Cloud, User } from 'hpe-icons/grommet';
+
+function App() {
+  return (
+    <Grommet theme={hpe} themeMode="auto" >
+      <Accessibility />
+      <Cloud size="xxlarge" />
+      <StatusWarning color="warning" />
+      <User size="small" color="onStrong" />
+    </Grommet>
+  );
+}
+```
+
+### Available Props
+
+All Grommet icon components support the following props:
+
+- `size`: `'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge'` or custom size string
+- `color`: Any color from your theme or CSS color value
+- `a11yTitle`: Accessibility label for screen readers
+
+### Theming
+
+Icons respect Grommet's theming system. All icons respect semantic icon colors provided by `hpe-design-tokens` and/or `grommet-theme-hpe`.
+
+## Available Icons
+
+### Categories
+
+The library includes icons across multiple categories:
+
+- **Interface**: Navigation, actions, controls
+- **Technology**: Cloud, servers, networking, security
+- **Business**: Analytics, documents, communication
+- **Users**: People, roles, permissions
+- **Media**: Audio, video, images
+- **Status**: Success, warning, error indicators
+- **And many more...**
+
+### Complete Icon List
+
+For a complete list of all 450+ available icons, explore the Storybook documentation or check the `public/img/` directory.
+
+## Development
+
+### Running Storybook
+
+To explore all available icons and their usage:
+
+```bash
+yarn storybook
 ```
 
-You can also install [eslint-plugin-react-x](https://github.com/Rel1cx/eslint-react/tree/main/packages/plugins/eslint-plugin-react-x) and [eslint-plugin-react-dom](https://github.com/Rel1cx/eslint-react/tree/main/packages/plugins/eslint-plugin-react-dom) for React-specific lint rules:
-
-```js
-// eslint.config.js
-import reactX from 'eslint-plugin-react-x'
-import reactDom from 'eslint-plugin-react-dom'
-
-export default tseslint.config([
-  globalIgnores(['dist']),
-  {
-    files: ['**/*.{ts,tsx}'],
-    extends: [
-      // Other configs...
-      // Enable lint rules for React
-      reactX.configs['recommended-typescript'],
-      // Enable lint rules for React DOM
-      reactDom.configs.recommended,
-    ],
-    languageOptions: {
-      parserOptions: {
-        project: ['./tsconfig.node.json', './tsconfig.app.json'],
-        tsconfigRootDir: import.meta.dirname,
-      },
-      // other options...
-    },
-  },
-])
+### Building
+
+```bash
+yarn build
 ```
+
+### Scripts
+
+- `yarn dev` - Start development server
```

Comment:
Does `yarn dev` run anything valuable or should we exclude this command?

Reply:
removed